addons/l10n_ua/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CompanyUA(models.Model):
+    _inherit = 'res.company'
+
+    company_registry = fields.Char(related='partner_id.company_registry')
```

Comment:
What is the use of this field? It does not seems to be used. Also by convention fields added from localization needs to be prefixed l10n_xx_